apps/weshipit/pages/react-native-tools/[slug].tsx

```diff
@@ -19,12 +25,12 @@ export function Slug({ records, recomendedRecords }) {
 
   return (
     <Layout
-      seoTitle="Slug React Native Serverless API"
-      seoDescription="The best tools and resources for busy developers"
       ogImageTitle="React Native Serverless API"
       withAccessoryRight={
         <Button href={linksApi.airtable.TOOLS_FORM}>Add a new API</Button>
       }
+      seoTitle={'Slug React Native Serverless API'}
```

Comment:
instead of duplicating the content you can use `metadata.title` and `metadata.description`.